benefits/static/css/admin/styles.css

```diff
@@ -4,10 +4,10 @@
 /* Primary Button: Use all three classes: btn btn-lg btn-primary */
 /* Outline Primary Button: Use all three classes: btn btn-lg btn-outline-primary */
 /* Set button width in parent with Bootstrap column */
-/* Height: 60px on Desktop; 72 on mobile*/
+/* Height: 48px (3rem) on Desktop; 72 on mobile*/
 
 :root {
-  --primary-button-padding: 19px 0;
+  --primary-button-padding: 1rem 0;
```

Comment:
This took me a second to parse... 

The above change to say `48px (3rem)` is because of `1rem` of padding on top and bottom, plus `16px` font size (=`1rem`) for a total of `3rem` -- is that right?

Would you mind updating this comment to indicate how `3rem` is calculated? Since that value never shows up in the code.

Reply:
@thekaveman added a lot more clarifying comments. 

Once these buttons are *official* (as in, part of the new #2648 designs), the plan is to move the button code into `theme.css`. This is just an interim place.